### ISSUE TITLE
Update default content/tours with new docs URL

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#144]
~~Blocked by #143.~~ 

New URL: https://docs.sitenow.uiowa.edu

### Default Content Files
https://github.com/uiowa/uiowa/blob/f6df2a4b48e0e663f06c877845502cf755db7790/docroot/profiles/custom/sitenow/content/paragraph/1763c09d-891e-48b0-a8c9-15df74023ed5.json#L120

find: uiowa.github.io\/sitenow
replace: docs.sitenow.uiowa.edu

### Tour files
https://github.com/uiowa/uiowa/blob/5e741fa6644389cf7dce96269231ce82615aced9/docroot/profiles/custom/sitenow/config/sync/tour.tour.site-settings.yml#L20

find: https://uiowa.github.io/sitenow
replace: https://docs.sitenow.uiowa.edu

**Comment by joewhitsitt** [comment]
I know @ethanstjohn had a branch started for this locally but this needs to get in sooner than later so #160 was created to solve this and another issue.
